eclipse/mars/gcp-eclipse-mars.target

```diff
@@ -25,8 +25,8 @@
       <repository location="http://download.eclipse.org/releases/mars/"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.epp.logging.aeri.feature.feature.group" version="2.0.3.v20160923-1310"/>
-      <unit id="org.eclipse.epp.logging.aeri.feature.source.feature.group" version="2.0.3.v20160923-1310"/>
+      <unit id="org.eclipse.epp.logging.aeri.feature.feature.group" version="2.0.4.v20170307-1435"/>
+      <unit id="org.eclipse.epp.logging.aeri.feature.source.feature.group" version="2.0.4.v20170307-1435"/>
```

Comment:
You didn't change this for Neon or Oxygen. (They still have entries with "2.0.3".) Is this intended?